Content.Shared/Lathe/LatheComponent.cs

```diff
@@ -89,6 +88,22 @@ public sealed class LatheComponent : Component
         #endregion
     }
 
+    [RegisterComponent, NetworkedComponent]
+    public sealed class EmagLatheRecipesComponent : Component
+    {
+        /// <summary>
+        /// All of the dynamic recipes that the lathe is capable to get using EMAG
+        /// </summary>
+        [DataField("emagDynamicRecipes", customTypeSerializer: typeof(PrototypeIdListSerializer<LatheRecipePrototype>))]
+        public readonly List<string> EmagDynamicRecipes = new();
+
+        /// <summary>
+        /// All of the static recipes that the lathe is capable to get using EMAG
+        /// </summary>
+        [DataField("emagStaticRecipes", customTypeSerializer: typeof(PrototypeIdListSerializer<LatheRecipePrototype>))]
+        public readonly List<string> EmagStaticRecipes = new();
+    }
+
```

Comment:
Separate file please, also none of the recipes are networked (use AutoNetworkedField / AutoGenerateComponentState is probably easiest).